app/components/failed_network_action/index.js

```diff
@@ -15,25 +15,32 @@ export default class FailedNetworkAction extends PureComponent {
     static propTypes = {
         onRetry: PropTypes.func,
         theme: PropTypes.object.isRequired,
-        errorTitle: PropTypes.object,
-        errorDescription: PropTypes.object,
-    };
-
-    static defaultProps = {
-        errorTitle: {
-            id: t('mobile.failed_network_action.title'),
-            defaultMessage: 'No internet connection',
-        },
-        errorDescription: {
-            id: t('mobile.failed_network_action.description'),
-            defaultMessage: 'There seems to be a problem with your internet connection. Make sure you have an active connection and try again.',
-        },
     };
 
     render() {
         const {theme, onRetry} = this.props;
         const style = getStyleFromTheme(theme);
 
+        const errorTitle = {
+            id: t('mobile.failed_network_action.title'),
+            defaultMessage: 'No internet connection',
+        };
+
+        const errorDescription = {
+            id: t('mobile.failed_network_action.shortDescription'),
+            defaultMessage: 'Messages will load when you have an internet connection or {refresh}.',
```

Comment:
maybe the id and defaultMessage can be a prop too? that way we can change the message depending on the screen you are seeing?

I'm thinking on this props.
action: PropTypes.func -> this is the action that can be taken
actionId: PropTypes.string -> the translation id for the action
actionDefaultMessage: PropTypes.string -> the default message used in the translation for the action
errorId: PropTypes.string -> the id for the translation
errorDefaultMessage: PropTypes.string -> the default message used in the translation for the message.

Reply:
Done @enahum